cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-detailed-ui.adoc

```diff
@@ -0,0 +1,290 @@
++
+[source,terminal]
+----
+rosa create ocm-role --mode auto --admin --yes
+----
++
+This command creates the OCM role and associates it with your Red Hat account.
++
+.Example output
++
+[source,terminal]
+----
+I: Creating ocm role
+I: Creating role using 'arn:aws:iam::000000000000:user/rosa-user'
+I: Created role 'ManagedOpenShift-OCM-Role-12561000' with ARN 'arn:aws:iam::000000000000:role/ManagedOpenShift-OCM-Role-12561000'
+I: Linking OCM role
+I: Successfully linked role-arn 'arn:aws:iam::000000000000:role/ManagedOpenShift-OCM-Role-12561000' with organization account '1MpZfntsZeUdjWHg7XRgP000000'
+----
+
+. Click *Next*.
+
+=== Other OCM role creation options
+* *Manual mode:* If you prefer to execute the AWS CLI commands yourself, you can define the mode as `manual` rather than `auto`. The CLI will output the AWS commands and the relevant JSON files are created in the current directory.
++
+Use the following command to create the OCM role in manual mode:
++
+[source,terminal]
+----
+rosa create ocm-role --mode manual --admin --yes
+----
+* *Basic OCM role:* If you prefer that the OCM has read only access to the account, create a basic OCM role. You will then need to manually create the required roles, policies, and OIDC provider using the CLI.
++
+Use the following command to create a Basic OCM role:
++
+[source,terminal]
+----
+rosa create ocm-role --mode auto --yes
+----
+
+== Creating an OCM user role
+
+As defined in the xref:../../../rosa_architecture/rosa-sts-about-iam-resources.html#rosa-sts-understanding-user-role_rosa-sts-about-iam-resources[user role documentation], the user role needs to be created so that ROSA can verify your AWS identity. This role has no permissions, and it is only used to create a trust relationship between the installer account and your OCM role resources.
+
+. Run the following command to create the user role and to link it to your Red Hat account:
++
+[source,terminal]
+----
+rosa create user-role --mode auto --yes
+----
++
+.Example output
++
+[source,terminal]
+----
+I: Creating User role
+I: Creating ocm user role using 'arn:aws:iam::000000000000:user/rosa-user'
+I: Created role 'ManagedOpenShift-User-rosa-user-Role' with ARN 'arn:aws:iam::000000000000:role/ManagedOpenShift-User-rosa-user-Role'
+I: Linking User role
+I: Successfully linked role ARN 'arn:aws:iam::000000000000:role/ManagedOpenShift-User-rosa-user-Role' with account '1rbOQez0z5j1YolInhcXY000000'
+----
++
+[NOTE]
+====
+As before, you can define `--mode manual` if you'd prefer to execute the AWS CLI commands yourself. The CLI outputs the AWS commands and the relevant JSON files are created in the current directory. Make sure to link the role.
+====
+
+. Click *Ok*.
+
+== Confirming successful account association
+
+. You will be brought back to the original window. You should now see your AWS account in the drop down menu. If you see your account, account association was successful.
```

Comment:
This statement need the correction according to the steps from the drawer. i.e

-  "You will be brought back to the original window" not required as you already in the same window.
-  Specify where should user see  the AWS account. Suggestion is "You should now see your AWS account in the "Associated AWS infrastructure account " drop down menu. If you see your account, account association was successful.